app/public/login/template.hbs

```diff
@@ -1,4 +1,4 @@
-<form {{action login model on="submit"}}>
+<form class="background" {{action login model on="submit"}}>
```

Comment:
Let's call this `login-form` instead so that we can remember what is being styled